data-es6.js

```diff
@@ -16662,6 +16665,7 @@ exports.tests = [
         return set instanceof S && set.has(123);
       */},
       res: {
+        closure:     true,
```

Comment:
Hmm, interesting solution. So closure forced uses polyfills for builtins subclassing... Looks like need extend tests for that.